etl/steps/data/meadow/ophi/2022-12-13/multidimensional_poverty_index.meta.yml

```diff
@@ -0,0 +1,114 @@
+dataset:
+  namespace: ophi
+  version: 2022-12-13
+  title: Multidimensional Poverty Index (MPI) (OPHI, 2022)
+  publication_year: 2022
+  short_name: multidimensional_poverty_index
+  description: |
+    The global Multidimensional Poverty Index (MPI) is an international measure of acute multidimensional poverty covering over 100 developing countries. It complements traditional monetary poverty measures by capturing the acute deprivations in health, education, and living standards that a person faces simultaneously.
```

Comment:
You have this description duplicated in snapshot metadata. In meadow step you call `convert_snapshot_metadata` which populates meadow dataset description with snapshot description, so there's no need to have it here. (No need to change it now, I didn't notice it got merged)

Reply:
Thanks, sometimes I feel I like more to have all metadata visible in each step, but it's true the other way is better for maintainability